src/components/matchMakingModal/selfMatchModal.module.css

```diff
@@ -0,0 +1,66 @@
+.selfMatchModal{

```

Comment:
Change directory and file name to be UpperCamelCase

Reply:
ok